keps/sig-network/2091-admin-network-policy/README.md

```diff
@@ -0,0 +1,1215 @@
+- e2e test cases must cover various combinations of `namespaceSet*s` in each ingress/egress rule.
+- Ensure that namespace matching strategies work as expected.
+- Add unit tests to test the validation logic which shall be introduced for cluster-scoped policy resources.
+  - Ensure exactly one selector has to be set in an `Subject` section.
+  - Ensure exactly one selector has to be set in an `AdminNetworkPolicyPeer` section.
+  - Test cases for fields which are shared with NetworkPolicy, like `endPort` etc.
+- Ensure that only administrators or assigned roles can create/update/delete cluster-scoped policy resources.
+- Ensure smooth integration with existing Kubernetes NetworkPolicy.
+  - Ensure all positive priority ANP rules are evaluated before any NetworkPolicy rules. 
+  - Ensure ANP rules with priority="0" are evaluated after any NetworkPolicy rules.
+
+### Graduation Criteria
+
+#### Alpha to Beta Graduation
+
+- Gather feedback from developers and surveys
+- At least 2 CNI providers must provide a functional and scalable implementation 
+  for the complete set of alpha features.
+    - Specifically,  ensure that only selecting E/W cluster traffic is plausible 
+      at scale 
+- Evaluate the need for multiple `Subject`s per ANP 
+- Evaluate "future work" items based on feedback from community and challenges 
+  faced by implementors.
+- Ensure extensibility of adding new fields. i.e. adding new fields do not "fail-open" 
+  traffic for older clients. 
+
+#### Beta to GA Graduation
+
+- At least 4 CNI providers must provide a scalable implementation for the complete set
+  of beta features
+- More rigorous forms of testing
+  — e.g., downgrade tests and scalability tests
+- Allowing time for feedback
+- Completion of all accepted "future work" items
+
+### Upgrade / Downgrade Strategy
+
+<!--
+If applicable, how will the component be upgraded and downgraded? Make sure
+this is in the test plan.
+
+Consider the following in developing an upgrade/downgrade strategy for this
+enhancement:
+- What changes (in invocations, configurations, API use, etc.) is an existing
+  cluster required to make on upgrade, in order to maintain previous behavior?
+- What changes (in invocations, configurations, API use, etc.) is an existing
+  cluster required to make on upgrade, in order to make use of the enhancement?
+-->
+
+#### Upgrade considerations
+
+As such, the cluster-scoped policy resources are new and shall not exist prior
+to upgrading to a new version. Thus, there is no direct impact on upgrades.
+
+#### Downgrade considerations
+
+Downgrading to a version which no longer supports cluster-scoped policy APIs
+must ensure that appropriate security rules are created to mimick the cluster-scoped
+policy rules by other means, such that no unintended traffic is allowed, and all
+intended traffic is allowed.
+
+### Version Skew Strategy
+
+n/a
+
+## Production Readiness Review Questionnaire
```

Comment:
I believe some questions are missing - for example questions on how the admin and the user know it is working. Is that because they don't apply at alpha or because you have an old copy of the questions? Can you check please?

Reply:
Yeah I checked, this PR is so OLD that the PRR questionnaire has been updated :) I'll bump it